eth_crowdfund_be/eth_crowdfund_api/manage.py

```diff
@@ -0,0 +1,18 @@
+import os 
+from flask_script import Manager 
+from flask_migrate import Migrate, MigrateCommand 
+from src.models import Campaign, Request
+
+from src.app import create_app, db 
+
+env_name = os.getenv('FLASK_ENV')
+app = create_app(env_name)
+
+migrate = Migrate(app=app, db=db)
```

Comment:
the mangae file is responsible for updating the database. for example, to run a migration the command is `python manage.py db migrate`